test/stablehlo/test_stablehlo_compile.py

```diff
@@ -0,0 +1,38 @@
+import torch_xla
+import torch_xla.core.xla_model as xm
+import torch
+import torchvision
+import unittest
+import torch_xla.debug.metrics as met
+import torch_xla.debug.metrics_compare_utils as mcu
+import numpy as np
+
+
+class StableHloCompileTest(unittest.TestCase):
+
+  def test_resnet18_stablehlo_compile(self):
+    resnet18 = torchvision.models.resnet18()
+    resnet18.eval()
+    np_input = np.random.randn(4, 3, 224, 224)
+    torch_input = torch.tensor(np_input).float()
+    cpu_output = resnet18(torch_input)
+    # Run ResNet on XLA device.
+    device = xm.xla_device()
+    # materalize the fake data for test purpose
+    xm.mark_step()
```

Comment:
I wonder why we need to do mark_step. The comment says "materalize the fake data for test purpose" but we don't have a HLO graph at this point.

Reply:
I was referencing the code in [dynamo test](https://github.com/pytorch/xla/blob/master/test/dynamo/test_dynamo.py). I this the purpose is to clear the graph from previous tests.